react/features/toolbox/components/web/Toolbox.tsx

```diff
@@ -97,7 +97,9 @@ export default function Toolbox({
     const toolbarButtonsToUse = toolbarButtons || reduxToolbarButtons;
     const chatOpen = useSelector((state: IReduxState) => state['features/chat'].isOpen);
     const isDialogVisible = useSelector((state: IReduxState) => Boolean(state['features/base/dialog'].component));
-    const jwtDisabledButtons = useSelector(getJwtDisabledButtons);
+    const jwt = useSelector((state: IReduxState) => state['features/base/jwt'].jwt);
+    const localParticipant = useSelector(getLocalParticipant);
+    const jwtDisabledButtons = getJwtDisabledButtons(jwt, localParticipant?.features);
```

Comment:
Ah I see now, by not using a selector for the whole thing we'll only call this when the JWT or the local participant changes, right?

Reply:
Exactly!